library/std/convert.qs

```diff
@@ -264,4 +264,20 @@ namespace Microsoft.Quantum.Convert {
         );
     }
 
+    /// # Summary
+    /// Produces string representation of given double with desired precision (performs rounding).
```

Comment:
Looking at the similar API documents at https://github.com/microsoft/qsharp-runtime/blob/da1872db685826861313ef078acab615e04f1334/src/Simulation/QSharpFoundation/Convert/Convert.qs#L197, I'd suggest to phrase this a bit smoother:
```suggestion
    /// Converts a given double-precision floating-point number to a string representation with desired precision, rounding if required.
```